lib/org/chromium/webidl/ast/Definition.js

```diff
@@ -18,9 +20,15 @@ foam.CLASS({
       name: 'id',
       factory: function() { return this.definition.getName(); },
     },
+    {
+      name: 'sources',
+    },
```

Comment:
This is not a typo. This was done because I would like the top level entity (the definition) to contain a list of all the files that were referenced.